token/token.go

```diff
@@ -419,6 +419,20 @@ func (mgm *tokenManager) initServiceAccountToken(req *goa.RequestData) (string,
 	return mgm.serviceAccountToken, nil
 }
 
+func (mgm *tokenManager) IsServiceAccount(ctx context.Context, serviceAccountUser string) bool {
```

Comment:
I've implemented this function in https://github.com/fabric8-services/fabric8-auth/blob/master/token/token.go#L438
You can re-use it now.